src/wp-content/themes/twentynineteen/sass/blocks/_blocks.scss

```
@@ -950,6 +950,11 @@
 		}
 	}
 
+	//! wp-block-social-links
```

```suggestion
	//! Social Icons
```

Use the block name for the comment.